api_tests/e2e/rfc003/btc_eth-erc20/refund_beta.ts

```diff
@@ -0,0 +1,183 @@
+    await alice.wallet.eth().fund("1");
+
+    let deployReceipt = await tobyWallet
+        .eth()
+        .deployErc20TokenContract(global.project_root);
+    let tokenContractAddress: string = deployReceipt.contractAddress;
+
+    let swapRequest: SwapRequest = {
+        alpha_ledger: {
+            name: "bitcoin",
+            network: "regtest",
+        },
+        beta_ledger: {
+            name: "ethereum",
+            network: "regtest",
+        },
+        alpha_asset: {
+            name: "bitcoin",
+            quantity: alphaAssetQuantity.toString(),
+        },
+        beta_asset: {
+            name: "erc20",
+            quantity: betaAssetQuantity.toString(),
+            token_contract: tokenContractAddress,
+        },
+        beta_ledger_redeem_identity: aliceFinalAddress,
+        alpha_expiry: alphaExpiry,
+        beta_expiry: betaExpiry,
+        peer: bobComitNodeAddress,
+    };
+
+    let bobWalletAddress = await bob.wallet.eth().address();
+
+    let mintReceipt = await ethereum.mintErc20Tokens(
+        tobyWallet.eth(),
+        tokenContractAddress,
+        bobWalletAddress,
+        bobInitialErc20
+    );
+    mintReceipt.status.should.equal(true);
+
+    let erc20Balance = await ethereum.erc20Balance(
+        bobWalletAddress,
+        tokenContractAddress
+    );
+
+    erc20Balance.eq(bobInitialErc20).should.equal(true);
+
+    let aliceErc20BalanceBefore: BN = await ethereum.erc20Balance(
+        aliceFinalAddress,
+        tokenContractAddress
+    );
+
+    const actions = [
+        {
+            actor: bob,
+            action: ActionKind.Accept,
+            requestBody: {
+                beta_ledger_refund_identity: bob.wallet.eth().address(),
+            },
+            state: (state: any) => state.communication.status === "ACCEPTED",
+        },
+        {
+            actor: alice,
+            action: ActionKind.Fund,
+            state: (state: any) => state.alpha_ledger.status === "Funded",
+        },
+        {
+            actor: bob,
+            action: ActionKind.Deploy,
+            state: (state: any) => state.beta_ledger.status === "Deployed",
+        },
+        {
+            actor: bob,
+            action: ActionKind.Fund,
+            state: (state: any) => state.beta_ledger.status === "Funded",
+            test: {
+                description:
+                    "[bob] Should have less beta asset after the funding",
+                callback: async () => {
+                    let bobErc20BalanceAfter = await ethereum.erc20Balance(
+                        bob.wallet.eth().address(),
+                        tokenContractAddress
+                    );
+
+                    bobErc20BalanceAfter
+                        .lt(bobInitialErc20)
+                        .should.be.equal(true);
+                },
+            },
+        },
+        {
+            actor: bob,
+            test: {
+                description: "[bob] Is waiting for beta htlc to expire",
```

Comment:
🤡 since we've already said it's `bob`, maybe the test framework can handle the `[Bob]` for us?